### DISCUSSION TITLE
completely broken styling after uploading to live server

### DISCUSSION THREAD

**Discussion by bramvanderkooij** [rappasoft/laravel-livewire-tables#1314] [Q&A]
![example 1](https://github.com/rappasoft/laravel-livewire-tables/assets/62700508/93e31dad-71d5-47f6-b23c-062115867a16)
![example 2](https://github.com/rappasoft/laravel-livewire-tables/assets/62700508/8d50c983-9848-4d64-b29e-70f75daeeb76)

**Comment by lrljoe** [comment]
You need to make sure you run your bundler updates if you're using Vite/Mix

**Reply by bramvanderkooij** [reply]
I am using Vite, I'm also a total beginner to actually putting stuff on a server, could you elaborate on how bundling works?

**Reply by lrljoe** [reply]
Suggest you check out the official Laravel documentation on asset bundling as it's quite a complex subject.

Ultimately you'll need to run
npm i
npm run build

on your production server in your project root to install the relevant dependencies (including dev dependencies), and generate the proper css/js files

The ideal would be to deploy with prebuilt files, but this will work in the interim.
